server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -154,6 +157,7 @@ void restoreShouldAppendMessageToMailboxWhenMatchingSubjectContains() throws Exc
 
                 String taskId =
                     given()
+                        .queryParam("action", Action.RESTORE.name())
```

Comment:
please put the raw string instead of the reference to the enum.
The goal is that the tests actually break if someone refactor the RESTORE enum value because it would change the API.